pkg/sql/crdb_internal.go

```diff
@@ -1932,10 +1932,18 @@ CREATE VIEW crdb_internal.cluster_contended_indexes (
     crdb_internal.tables,
     crdb_internal.table_indexes
   WHERE
-    crdb_internal.cluster_contention_events.index_id
-    = crdb_internal.table_indexes.index_id
-    AND crdb_internal.cluster_contention_events.table_id
-      = crdb_internal.tables.table_id
+    (
+      crdb_internal.cluster_contention_events.index_id
+      = crdb_internal.table_indexes.index_id
+      AND crdb_internal.cluster_contention_events.table_id
+        = crdb_internal.table_indexes.descriptor_id
+    )
+    AND (
+        crdb_internal.cluster_contention_events.table_id
+        = crdb_internal.tables.table_id
+      )
+  ORDER BY
+    num_contention_events DESC
```

Comment:
Hmm Sorry that i'm a bit late to this. Are these parenthesis around the `WHERE` clause necessary ?

Reply:
Oops, no. #72929.